dev-infrastructure/templates/aks-development.bicep

```diff
@@ -270,56 +227,59 @@ resource aksCluster 'Microsoft.ContainerService/managedClusters@2022-03-01' = {
         minCount: agentMinCount
         maxCount: agentMaxCount
         vmSize: agentVMSize
-        vnetSubnetID: vnetSubnetId
-        podSubnetID: podSubnetId
+        vnetSubnetID: aksNodeSubnet.id
+        podSubnetID: aksPodSubnet.id
         maxPods: maxPods
       }
     ]
     networkProfile: {
```

Comment:
@mjlshen Also Microsoft [suggested](https://docs.google.com/document/d/1D13mcDkH9xK3m5RInzavF-AX3OGmWpsj6fh_sUzd_CI/edit?disco=AAABHffx14M) to use cilium as dataplane plugin and we were using it for perfscale runs, it would be better if we follow that recommendations for dev environment as well.

Reply:
Thanks for letting me know! Azure CNI is actually Cilium I think https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium, but to your point I need to enable network policy